models/src/main/protobuf/CasperMessage.proto

```diff
@@ -229,6 +235,7 @@ message PeekProto {
 message BondProto {
   bytes validator = 1;
   int64 stake     = 2;
+  bool  isActive  = 3;
```

Comment:
We cannot set this on previous blocks so after decoding this field will have default value, which is for _bool_ `false` .
https://developers.google.com/protocol-buffers/docs/proto3#default

Maybe it is sufficient to call it something like `isSlashed` or `deactivated` and can be backward compatible.

Reply:
Do you think we need `isSlashed` field?